app/views.py

```diff
@@ -278,6 +280,24 @@ def post(self, request, *args, **kwargs):
         return redirect('conditions', contract_id, installment_id)
 
 
+class ConditionBackupProofView(View):
+
+    def post(self, request, *args, **kwargs):
+        contract_id = self.kwargs.get('contract_id')
+        installment_id = self.kwargs.get('installment_id')
+
+        condition_id = self.kwargs.get('condition_id')
+        condition = InstallmentCondition.objects.get(pk=condition_id)
+        try:
```

Comment:
we are only handling ValidationError, which is raised by invalid file extensions. we should also handle errors related to dropbox access.